config/routes.rb

```diff
@@ -137,6 +137,7 @@ def external_redirect(path_prefix, target)
     resources :ministerial_roles, path: 'ministers', only: [:index, :show], localised: true
     resources :people, only: [:index, :show], localised: true
 
+    # TODO: Remove `:show` when policy group paths can be otherwise generated
```

Comment:
https://trello.com/c/wvdvAKek/19-create-a-mechanism-to-provide-url-and-path-helpers-for-migrated-formats-without-routes